internal/pkg/mapper/orchestrator/local_load_test.go

```diff
@@ -313,43 +312,3 @@ invalid orchestrator config "branch/other/orchestrator":
 `
 	assert.Equal(t, strings.Trim(expectedError, "\n"), err.Error())
 }
-
-func createLocalLoadFixtures(t *testing.T, context mapper.Context) *model.ConfigState {
```

Comment:
V `mappers` pkg mame rozne metody na vytvaranie fixtures, `create*` ...
Pouzivaju sa jednym alebo viacerými testami.

Presunul som ich vsetky vzdy do hlavneho suboru balicka, `*_test.go`.
V dalsich PR v nich budu drobne zmeny, a takto je to prehladnejsie a jednotne.

Kod sa nemenil iba presuval medzi subormi.